MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2226,13 +2226,14 @@ public string GetMapArgs(EncodingJobInfo state)
             }
 
             var args = string.Empty;
+            int videoStreamIndex = state.MediaSource.MediaStreams.Where(i => i.Path == state.VideoStream.Path).ToList().IndexOf(state.VideoStream);
```

Comment:
Given that you are now using this so much throughout the class, I think it should be a static method eg:

```C#
        private static int FindIndex(IReadOnlyList<MediaStream> mediaStreams, MediaStream streamToFind)
        {
            var index = 0;
            var length = mediaStreams.Count;
            for (var i = 0; i < length; i++)
            {
                var currentMediaStream = mediaStreams[i];
                if (currentMediaStream.Path == streamToFind.Path)
                {
                    index++;
                }

                if (currentMediaStream == streamToFind)
                {
                    return index;
                }
            }

            return -1;
        }
```